src/js/plugin-bridge/PluginModules.js

```diff
@@ -74,7 +74,8 @@ module.exports = {
     TabForm: 'TabForm',
     ToggleButton: 'ToggleButton',
     Typeahead: 'Typeahead',
-    UserDropup: 'UserDropup'
+    UserDropup: 'UserDropup',
+    'form-elements': 'form/index'
```

Comment:
The dash-case looks a little out of place, but I guess it makes sense 🤔 
Let's roll with it 💪